packages/contracts/contracts/interfaces/ISmartInvoiceInstant.sol

```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+interface ISmartInvoiceInstant {
```

Comment:
This should inherit from ISmartInvoice, no?

Reply:
Yes it will once the contract upgrade PR is merged. To avoid potential merge conflict and to show only the pay now addition, I opted to do this for now. The proper inheritance and other changes are in queue.